src/Symfony/Component/Validator/ConstraintViolation.php

```diff
@@ -84,7 +84,7 @@ public function __toString()
             $class .= '.';
         }
 
-        if ('' !== $code = (string) $this->code) {
+        if (null !== $this->code && '' !== $code = $this->code) {
```

Comment:
Note: I've tweaked this line to avoid a PHP notice in f03bf95941d97d5e607a6b33eec0391480ef5290

Reply:
Thank you @ogizanagi 👍